apps/docs/src/guide/quickstart/index.md

```diff
@@ -0,0 +1,87 @@
+# Quickstart
+
+You can quickly bootstrap a full-stack Fuel project locally with the following command:
+
+::: code-group
+
+```sh [pnpm]
+pnpm create fuels
```

Comment:
Does it need the project name?

This is the output that I am receiving

<img width="602" alt="Screenshot 2023-12-18 at 18 45 16" src="https://github.com/FuelLabs/fuels-ts/assets/30977845/3fed8587-eb19-4128-a1dc-274b7736474e">

I had to run `pnpm create fuels myProject` to make this work

Reply:
Please run the command with the pr release tag like I mentioned here: https://fuellabs.slack.com/archives/C04LVA1UF7X/p1702913792062199

If you don't use the pr release tag, it uses the first, old cached version